futurex_openedx_extensions/dashboard/serializers.py

```diff
@@ -143,6 +143,25 @@ def _get_name(self, obj: Any, alternative: bool = False) -> str:
 
         return alt_name if alternative else full_name
 
+    def _get_arabic_name(self, obj: Any) -> str:
+        """
+        Get arabic name of user
+
+        :param obj: The user object.
+        :type obj: Any
+        :return: The arabic name of user.
+        """
+
+        arabic_first_name = self._get_extra_field(obj, 'arabic_first_name')
+        arabic_last_name = self._get_extra_field(obj, 'arabic_last_name')
+        arabic_full_name = arabic_first_name or arabic_last_name
+        if arabic_first_name and arabic_last_name and not arabic_first_name == arabic_last_name:
+            arabic_full_name = ' '.join(filter(None, (arabic_first_name, arabic_last_name)))
+        arabic_name = self._get_extra_field(obj, 'arabic_name')
+
+        alt_name = arabic_full_name or arabic_name or ''
+        return alt_name.strip()
```

Comment:
Thank you @tehreem-sadat . I recommend using `arabic_name` then the other two fields. I can see that our data doesn't have `first` and `last` for most records
```suggestion
        arabic_name = (self._get_extra_field(obj, 'arabic_name') or '').strip()
        if arabic_name:
            return arabic_name

        # the first-name last-name logic.....
```